### PR TITLE
Support forceBypass under armingModes

### PR DESCRIPTION
**Pull request by DouweM**
Closes https://github.com/node-alarm-dot-com/homebridge-node-alarm-dot-com/issues/76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ This is a plugin for Homebridge, allowing communication with Alarm.com endpoints
 - Locks
   - Lock/Unlock switch
 - Garage Doors
-  - Open/Close switch 
-- Thermostats 
-  - Set mode Off/Heat/Cool/Auto 
+  - Open/Close switch
+- Thermostats
+  - Set mode Off/Heat/Cool/Auto
```
Comment (chase9) [inline README.md:33]: Could you remove this file from your PR? It rather not have whitespace changes in the merge. Thanks!

**Review by chase9 — CHANGES_REQUESTED** [review]

**Comment by chase9** [comment]
Thanks for submitting this! Could you remove the README changes since they're just whitespace, and also let me know if you have tested this on your own instance?

**Comment by DouweM** [comment]
@chase9 Thanks for the review, I've removed the whitespace changes.

I've tested the forceBypass option directly on the API, but I didn't yet look into actually building and running and testing the plugin. What would be the easiest way to do that?

**Comment by DouweM** [comment]
@chase9 I had a chance to test it and confirmed it works as expected!

**Review by chase9 — APPROVED** [review]

**Comment by chase9** [comment]
Great, looks good and thanks for the PR!

**Pull request merged**